GetStream/stream-chat-android
# Using API's lastMessageAt instead of local one

### 🎯 Goal

Make local order follow API order

### 🎨 UI Changes

<table>
<thead>
<tr>
<th>Before</th>
<th>After</th>
</tr>
</thead>
<tbody>
<tr>
<td>
<video src="https://user-images.githubusercontent.com/10619102/168660445-bd7004b9-8105-4547-ba8b-af5b29bc13c4.mov" controls="controls" muted="muted" />
</td>
<td>
<video src="https://user-images.githubusercontent.com/10619102/168660465-63d1f03b-6f7a-40b8-9e6c-b84c4c69562a.mov" controls="controls" muted="muted" />
</td>
</tr>
</tbody>
</table>

### 🧪 Testing

- Add messages in a thread in a channel that is not the first one without internet connection
- Enable internet
- Check that the order doesn't change anymore after the channels are loaded. 

### ☑️Contributor Checklist

#### General

#### Code & documentation

### ☑️Reviewer Checklist

### 🎉 GIF
![giphy](https://user-images.githubusercontent.com/10619102/168676973-c6f3fb1a-d04b-4730-91e8-bfeead926651.gif)
_shuffling_
